other/materials_designer/specific_examples/heterostructure_high_k_metal_gate.ipynb

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
<p>We can split the two steps below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>